acceptance/tests/parser_functions/hiera/lookup_data.rb

```diff
@@ -32,7 +32,6 @@
       - "yaml"
     :logger: "console"
     :hierarchy:
-      - "%{fqdn}"
```

Comment:
Was `fqdn` not being used in the test?

Reply:
It apparently wasn't used in any of the `lookup_data.rb` tests, but the strict erroring was causing failures.